portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SyncAttributes.testcase

```diff
@@ -642,18 +642,18 @@ definition {
 		var attributesDataTypeAscending1 = '''
 			male,agreedToTermsOfUse,defaultUser,emailAddressVerified,
 			birthday,createDate,modifiedDate,createDate,modifiedDate,status,classNameId,classPK,contactId,
-			employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,companyId,companyId,contactId,facebookId,
-			ldapServerId,portraitId,timeZoneId,userId,emailAddress,employeeNumber,facebookSn,hoursOfOperation,
-			jabberSn,jobClass,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,firstName,
-			firstName,googleUserId,greeting,jobTitle,languageId,lastName,lastName,middleName,middleName,openId
+			employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,companyId,contactId,facebookId,
+			ldapServerId,portraitId,timeZoneId,userId,emailAddress,employeeNumber,facebookSn,firstName,hoursOfOperation,
+			jabberSn,jobClass,jobTitle,lastName,middleName,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,
+			firstName,googleUserId,greeting,jobTitle,languageId,lastName,middleName,openId
 		''';
 		var attributesDataTypeAscending2 = "screenName,uuid";
 		var attributesDataTypeDescending1 = '''
-			emailAddress,employeeNumber,facebookSn,hoursOfOperation,
-			jabberSn,jobClass,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,firstName,
-			firstName,googleUserId,greeting,jobTitle,languageId,lastName,lastName,middleName,middleName,openId,screenName,uuid,
+			emailAddress,employeeNumber,facebookSn,firstName,hoursOfOperation,
+			jabberSn,jobClass,jobTitle,skypeSn,smsSn,twitterSn,userName,comments,emailAddress,externalReferenceCode,
+			firstName,googleUserId,greeting,jobTitle,languageId,lastName,middleName,openId,screenName,uuid,
 			classNameId,classPK,contactId,employeeStatusId,parentContactId,prefixListTypeId,suffixListTypeId,
-			companyId,companyId,contactId,facebookId,ldapServerId,portraitId,timeZoneId,userId,status,birthday,createDate,
+			companyId,contactId,facebookId,ldapServerId,portraitId,timeZoneId,userId,status,birthday,createDate,
```

Comment:
Hey @joshitagaki, I think that after `classPK` it should be `companyId` and then `contactId`. Could you also check and update the list?

![image](https://user-images.githubusercontent.com/76956227/223575206-c0249d70-c773-42b7-9018-3095c4c4e2e1.png)